### DISCUSSION TITLE
Error: The deploy step encountered an error: The process '/usr/bin/git' failed with exit code 128 ❌

### DISCUSSION THREAD

**Discussion by shiwanjun1995** [JamesIves/github-pages-deploy-action#1560] [Q&A]
## Additional Comments

<img width="1429" alt="截屏2024-03-31 下午4 48 09" src="https://github.com/JamesIves/github-pages-deploy-action/assets/38515714/02f36520-ad11-4119-b195-1c4b3ba97cfe">

`name: Build and Deploy
on:
  push:
    branches:
      - main
      
jobs:
  build-and-deploy:
    runs-on: ubuntu-latest

    steps:
    - name: Checkout 🛎️
      uses: actions/checkout@v4

    - name: Install and Build
      run: |
        npm install
        npm run-script build

    - name: Deploy 🚀
      uses: JamesIves/github-pages-deploy-action@v4
      with:
        token: ${{ secrets.ACCESS_TOKEN }}
        branch: gh-pages
        folder: build
        clean: true`

**Comment by shiwanjun1995** [comment]
https://github.com/shiwanjun1995/cicd-basic/actions/runs/8496968889/job/23275008338
Bully, Could you help me? 
@JamesIves

**Comment by shiwanjun1995** [comment]
This is success:
https://github.com/shiwanjun1995/cicd-basic/actions/runs/8497085551
But that is error:
https://github.com/shiwanjun1995/cicd-basic/actions/runs/8497133316
I am confused, because i changed code only, not yaml.
So you can watch this, if you need any info, please note, thanks a lot.
@JamesIves

**Comment by sykhro** [comment]
Running into the same issue.

**Comment by lyh543-autra** [comment]
seems solved in v4.6.1. See https://github.com/JamesIves/github-pages-deploy-action/discussions/1589#discussioncomment-9478303
